go-runtime/ftl/ftltest/ftltest.go

```diff
@@ -3,18 +3,94 @@ package ftltest
 
 import (
 	"context"
+	"fmt"
+	"reflect"
 
+	cf "github.com/TBD54566975/ftl/common/configuration"
 	"github.com/TBD54566975/ftl/go-runtime/ftl"
 	"github.com/TBD54566975/ftl/go-runtime/modulecontext"
 	"github.com/TBD54566975/ftl/internal/log"
+	"github.com/alecthomas/types/optional"
 )
 
-// Context suitable for use in testing FTL verbs.
-func Context() context.Context {
+type DBType int32
+
+const (
+	DBTypePostgres = DBType(modulecontext.DBTypePostgres)
+)
+
+// Context suitable for use in testing FTL verbs with provided options
```

Comment:
super nitty nit: would you mind adding some example code to this comment so that folks don't need to grep around to learn how to use this? Basically exactly what you put in your PR description:

```
ctx := ftltest.Context (
        ftltest.WithConfig("default", "anonymous"),
        ftltest.WithSecret("example", "test123"),
        ftltest.WithDSN("db", ftltest.DBTypePostgres, "..."),
        ftltest.WithFakeVerb(time.Time, func(ctx context. Context, req time. TimeRequest) (time.TimeResponse, error) {
            return time.TimeResponse{Time: stdtime.Date(2021, 9, 1, 0, 0, 0, 0, stdtime.UTC)}, nil
        },
    )
```

Definitely don't need to include every single option in the example if you want to keep the comment shorter, either. Thanks :)